calico_cni/tests/unit/test_cni_plugin.py

```diff
@@ -174,11 +174,15 @@ def test_add_mainline(self, m_json):
         self.plugin._provision_veth = MagicMock(spec=self.plugin._provision_veth)
         self.plugin._provision_veth.return_value = endpoint
 
+        # Mock out environment
```

Comment:
I don't think you need to mock out the environment here - you can just use the actual `self.plugin.env` value.